src/include/extern.h

```diff
@@ -541,12 +569,21 @@ extern void __wt_ovfl_txnc_free(WT_SESSION_IMPL *session, WT_PAGE *page);
 extern int __wt_ovfl_track_wrapup(WT_SESSION_IMPL *session, WT_PAGE *page);
 extern int __wt_ovfl_track_wrapup_err(WT_SESSION_IMPL *session, WT_PAGE *page);
 extern int __wt_reconcile(WT_SESSION_IMPL *session, WT_REF *ref, WT_SALVAGE_COOKIE *salvage, uint32_t flags);
-extern uint32_t __wt_split_page_size(WT_BTREE *btree, uint32_t maxpagesize);
-extern int __wt_bulk_init(WT_SESSION_IMPL *session, WT_CURSOR_BULK *cbulk);
-extern int __wt_bulk_wrapup(WT_SESSION_IMPL *session, WT_CURSOR_BULK *cbulk);
-extern int __wt_bulk_insert_row(WT_SESSION_IMPL *session, WT_CURSOR_BULK *cbulk);
-extern int __wt_bulk_insert_fix(WT_SESSION_IMPL *session, WT_CURSOR_BULK *cbulk);
-extern int __wt_bulk_insert_var(WT_SESSION_IMPL *session, WT_CURSOR_BULK *cbulk);
+extern int __wt_rec_write_status(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_PAGE *page);
+extern int __wt_rec_write_init(WT_SESSION_IMPL *session, WT_REF *ref, uint32_t flags, WT_SALVAGE_COOKIE *salvage, void *reconcilep);
+extern void __wt_rec_destroy(WT_SESSION_IMPL *session, void *reconcilep);
+extern int __wt_rec_block_free( WT_SESSION_IMPL *session, const uint8_t *addr, size_t addr_size);
+extern int __wt_rec_update_move( WT_SESSION_IMPL *session, WT_BOUNDARY *bnd, WT_SAVE_UPD *supd);
+extern int __wt_rec_txn_read(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_INSERT *ins, WT_ROW *rip, WT_CELL_UNPACK *vpack, WT_UPDATE **updp);
```

Comment:
@agorrod `__rec_row_bnd_split_or_write` is another example, we're doing an additional function call per element on the page, just to decide we don't have to split? (I'd rather the `__rec_row_bnd_split_or_write` code stayed inlined, I think.)